alerts/neato.markdown

```diff
@@ -0,0 +1,11 @@
+---
+title: "New Neato Botvacs Do Not Support Existing API"
+created: 2021-12-20 13:27:00
+integrations:
+  - neato
+homeassistant: ">0.30"
+---
+
+New Neato Botvacs no longer use the existing API that the current integration supports. So far the Neato D8 is using a new app and has no known API.
```

Comment:
Can we clarify "New"?

Reply:
As of now I only know about the D8 but I think they have more Botvacs coming in 2022. The dev docs don't list the supported Botvac models. They show the supported "service version" but we only know that when a device attempts to use the API. So its a bit difficult to tell. The only tell tale sign as of now is which app you have to use to setup the vacuum. So I had mentioned the app in hopes that would make it more clear :)